src/fosslight_source/cli.py

```
@@ -459,6 +459,24 @@ def merge_results(
+    final_output_path: str,
+    publish: bool,
+    log: Optional[logging.Logger] = None,
+) -> None:
+    """Copy scan artifacts from temp dir, then always remove the temp directory."""
+    if not temp_output_path or not os.path.isdir(temp_output_path):
+        return
+    try:
+        if publish:
+            shutil.copytree(temp_output_path, final_output_path, dirs_exist_ok=True)
+        shutil.rmtree(temp_output_path)
+    except Exception as ex:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)